### PR TITLE
stream layer bug fixing

### PR DESCRIPTION
**Pull request by Jade-JadeH**
There're two parts in this PR

**Looks and feels**
1. Use the new UI system
2. The plane model is not taking advantage of the maskmap and the normal map and looks rather flat previously. 

**Bug fixing**
There are a few bugs related to the stream layer sample: 
1. The dropdown selection changes randomly
3. The dropdown selection is not correctly reflecting user's selection

Before:

before

After

after

### CODE REVIEW

**Review by avr757 — APPROVED** [review]
tested it out and worked great

**Comment by avr757** [comment]
I removed the api key in the scene.

File: samples_project/Assets/SampleViewer/Samples/StreamLayer/StreamLayerWebSocketSubscribe.cs [hunk 1]
```diff
@@ -397,8 +427,8 @@ private void FlightSelected()
                 var CameraLocation = ArcGISCamera.GetComponent<ArcGISLocationComponent>();
                 double Longitude = flightLocation.Position.X;
                 double Latitude = flightLocation.Position.Y;
-
-                ArcGISPoint NewPosition = new ArcGISPoint(Longitude, Latitude, FlightSpawnHeight, flightLocation.Position.SpatialReference);
+                double height = flightLocation.Position.Z + 1500;
```
Comment (ZackAllen) [inline samples_project/Assets/SampleViewer/Samples/StreamLayer/StreamLayerWebSocketSubscribe.cs:430]: Good fix!

**Review by ZackAllen — APPROVED** [review]

File: samples_project/Assets/SampleViewer/Samples/StreamLayer/StreamLayerWebSocketSubscribe.cs (hunk 1 above)
Comment (Jade-JadeH) [reply samples_project/Assets/SampleViewer/Samples/StreamLayer/StreamLayerWebSocketSubscribe.cs:430]: thanks for the hint!

**Pull request merged**